atd-vzd/views/view_fatalities.sql

```diff
@@ -0,0 +1,32 @@
+-- view of fatalities in jurisdiction ID 5 (austin full purpose) with crash attributes joined
+CREATE OR REPLACE VIEW view_fatalities AS (
```

Comment:
This is really not a big deal, so please disregard if you want. I'm wondering what you think about naming the view with something that's more focused on what the data inside is? Maybe something like `fatalities_in_afp` or maybe `ftt_fatalities`. I'm not sure these are the best names, so take this with a grain of salt.

Reply:
yeah that makes sense to me too. i was trying to keep in pattern with some of the naming of other views i saw in the db but theres not really a set convention, and specifying ftt or in afp would be more descriptive.